packages/frontend/src/pages/home/latest-campaigns-section/index.tsx

```diff
@@ -21,7 +21,7 @@ export const LatestCampaignsSection = () => {
         <div className="relative flex flex-col gap-16">
             {/* TODO: add i18n */}
```

Comment:
Is this TODO still applicable? Maybe we should move it below where there's a need for `i18n` (the "view all campaigns text").

Reply:
Oh ye, I guess we can directly solve it, I'll add the translation.

Reply:
Fixed in bd09652fd0d9567326d5ff6221b1c2ef5c9187b2.